Feedback/Block/FeedbackForm.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Tasks\Feedback\Block;
+
+use Magento\Framework\View\Element\Template;
+use Tasks\Feedback\Model\ResourceModel\Feedback\Collection;
+use \Magento\Framework\UrlInterface;
+/**
+ * Feedback Form Block
+ *
+ * @package Tasks\Feedback\Block
+ */
+class FeedbackForm extends Template
+{
+    /**
+     * @var Collection
+     */
+    private $collection;
+
+    /**
+     * @var \Magento\Framework\UrlInterface
+     */
+    protected $_urlBuilder;
+
+    /**
+     * FeedbackForm constructor.
+     *
+     * @param Template\Context $context
+     * @param Collection $collection
+     * @param \Magento\Framework\UrlInterface $urlBuilder
+     * @param array $data
+     */
+    public function __construct(
+        Template\Context $context,
+        Collection $collection,
+        UrlInterface $urlBuilder,
+        array $data = []
+    ) {
+        $this->collection = $collection;
+        parent::__construct($context, $data);
+        $this->_urlBuilder = $urlBuilder;
```

Comment:
```suggestion
        parent::__construct($context, $data);
        $this->collection = $collection;
        $this->_urlBuilder = $urlBuilder;
```
Initialize the parent constructor either at the start or at the end. Please avoid initialize ii in between like this.